controllers/manilashare_controller.go

```diff
@@ -323,26 +323,43 @@ func (r *ManilaShareReconciler) reconcileNormal(ctx context.Context, instance *m
 	//
 	// check for required OpenStack secret holding passwords for service/admin user and add hash to the vars map
 	//
-	ctrlResult, err := r.getSecret(ctx, helper, instance, instance.Spec.Secret, &configVars)
+	hash, ctrlResult, err := ensureSecret(
```

Comment:
Another ensureSecret that I don't believe is required.

Reply:
ack and +1 for the same reason of the scheduler code.

Reply:
See the comment I left in the scheduler controller